src/domUpdates.js

```diff
@@ -28,74 +29,91 @@ const domUpdates = {
     document.querySelector('#dropdown-name').innerText = this.currentUser.name.toUpperCase();
     document.querySelector('#dropdown-goal').innerText = `DAILY STEP GOAL | ${this.currentUser.dailyStepGoal}`;
     document.querySelector('#dropdown-email').innerText = `EMAIL | ${this.currentUser.email}`;
-    this.showLeaderBoard(); // CSS is finnicky on it
-  },
-
-  showLeaderBoard() {
-    let dropdownFriendsStepsContainer = document.querySelector('#dropdown-friends-steps-container');
-    let friendsStepsParagraphs = document.querySelectorAll('.friends-steps');
-    let friendsWeeklySteps = this.currentUser.findFriends(this.userRepository);
-    friendsWeeklySteps.forEach(friend => {
-      dropdownFriendsStepsContainer.innerHTML += `
-          <p class='dropdown-p friends-steps'>${friend.firstName} |  ${friend.weeklySteps}</p>`;
-    });
-    friendsStepsParagraphs.forEach(paragraph => {
-      if (friendsStepsParagraphs[0] === paragraph) {
-        paragraph.classList.add('green-text');
-      }
-      if (friendsStepsParagraphs[friendsStepsParagraphs.length - 1] === paragraph) {
-        paragraph.classList.add('red-text');
-      }
-      if (paragraph.innerText.includes('YOU')) {
-        paragraph.classList.add('yellow-text');
-      }
-    });
-  },
+    // this.showLeaderBoard(); // CSS is finnicky on it
+  },
+
+  // showLeaderBoard() {
+  //   let dropdownFriendsStepsContainer = document.querySelector('#dropdown-friends-steps-container');
+  //   let friendsStepsParagraphs = document.querySelectorAll('.friends-steps');
+  //   let friendsWeeklySteps = this.currentUser.findFriends(this.userRepository);
+  //   friendsWeeklySteps.forEach(friend => {
+  //     dropdownFriendsStepsContainer.innerHTML += `
+  //         <p class='dropdown-p friends-steps'>${friend.firstName} |  ${friend.weeklySteps}</p>`;
+  //   });
+  //   friendsStepsParagraphs.forEach(paragraph => {
+  //     if (friendsStepsParagraphs[0] === paragraph) {
+  //       paragraph.classList.add('green-text');
+  //     }
+  //     if (friendsStepsParagraphs[friendsStepsParagraphs.length - 1] === paragraph) {
+  //       paragraph.classList.add('red-text');
+  //     }
+  //     if (paragraph.innerText.includes('YOU')) {
+  //       paragraph.classList.add('yellow-text');
+  //     }
+  //   });
+  // },
 
 
   ///////////// STEPS DISPLAY SECTION /////////////////////////////////
 
   stepCardDisplay() {
     this.stepMainCardDisplay();
-    this.stepsInfoCard();
+    this.stepsInfoCard(this.todaysDate);
```

Comment:
I was thinking about changing `info` to `stats` but do you think that might be too many S's in terms of semantics? stepsStatsCard sounds a bit much

Reply:
Worth discussing today. I think stats makes more sense given what's being displayed.